tokio/src/stream/mod.rs

```diff
@@ -817,3 +817,16 @@ pub trait StreamExt: Stream {
 }
 
 impl<St: ?Sized> StreamExt for St where St: Stream {}
+
+/// Private utiity function used in `merge ` and `Chain`
```

Comment:
nit: Describe what the function is doing
```suggested
/// Merge the size hints from two streams.
```

Reply:
I updated doc comment.